bazel/foreign_cc/BUILD

```diff
@@ -91,8 +91,8 @@ envoy_cmake_external(
         "//conditions:default": [],
     }),
     postfix_script = select({
-        "//bazel:windows_x86_64": "cp -L $EXT_BUILD_ROOT/external/com_github_c_ares_c_ares/nameser.h $INSTALLDIR/include/nameser.h",
-        "//conditions:default": "",
+        "//bazel:windows_x86_64": "cp -L $EXT_BUILD_ROOT/external/com_github_c_ares_c_ares/src/lib/nameser.h $INSTALLDIR/include/nameser.h && cp -L $EXT_BUILD_ROOT/external/com_github_c_ares_c_ares/src/lib/ares_dns.h $INSTALLDIR/include/ares_dns.h",
+        "//conditions:default": "cp -L $EXT_BUILD_ROOT/external/com_github_c_ares_c_ares/src/lib/ares_dns.h $INSTALLDIR/include/ares_dns.h",
```

Comment:
Why is this now needed in non-Windows?

Reply:
There was a change in directory structure in `c-ares` (https://github.com/c-ares/c-ares/commit/0bf721cdd7f64b893c4ad015c341de680617411b). We're using macros from `ares_dns.h` in `dns_imp_test.cc` - this header file is no longer available after `make install`, so I had to make a "hack" in order to make it visible for us.